src/exchanges/bitfinex-client.js

```diff
@@ -9,7 +9,20 @@ const Level3Snapshot = require("../level3-snapshot");
 const Level3Update = require("../level3-update");
 
 class BitfinexClient extends BasicClient {
-  constructor({ wssPath = "wss://api.bitfinex.com/ws/2", watcherMs, l2UpdateDepth = 250 } = {}) {
+  /**
+   *
+   * @param {Object} params
+   * @param {Boolean} [params.enableEmptyHeartbeatEvents]       (optional, default false). if true, emits empty events for all channels on heartbeat events which includes the sequenceId.
+   * @param {String} [params.tradeMessageType]                  (optional, defaults to "tu"). one of "tu", "te", or "all". determines whether to use trade channel events of type "te" or "tu", or all trade events. see https://blog.bitfinex.com/api/websocket-api-update/.
```

Comment:
new constructor param `tradeMessageType` to specify which trade events to handle. it was previously "tu" and that's probably fine for most ppl, but now you can specify "te", "tu", or "all"